File: _프로그래머스/19. 문자열 다루기 기본.py
"""
* Title: 문자열 다루기 기본
https://school.programmers.co.kr/learn/courses/30/lessons/12918
"""
def solution(s):
    answer = True
    length = len(s)

    if not (length == 4 or length == 6):
        answer = False
    if i in range(0, length):
        num = ord(s[i]) - 48
        if num >= 10:
            answer = False

    return answer

File: _프로그래머스 js/19. 문자열 다루기 기본.js
/*
* Title: 문자열 다루기 기본
https://school.programmers.co.kr/learn/courses/30/lessons/12918
*/
const solution = (s) => {
  const strls = Array.from(s);
  const len = strls.length;
  let answer = true;

  if (len === 4 || len === 6) {
    return false;
  }

  for (let i = 0; i < strls.length; i++) {
    const cal = strls[i].charCodeAt(0) - 48;
    if (cal >= 10) {
      answer = false;
    }
  }

  return answer;
};

// false
console.log(solution('a234'));
